src/audio/midi/message.rs

```diff
@@ -0,0 +1,21 @@
+use crate::Result;
+use std::convert::TryFrom;
+use wmidi::MidiMessage;
+
+#[derive(Debug)]
+pub struct Message {
+    pub timestamp: u64,
+    pub message: MidiMessage<'static>,
```

Comment:
Any better ideas for names here?

Reply:
I don't think so. The issue I was trying to deal with here is that the type is normally `MidiMessage<'a>`, where `<'a>` refers to the lifetime of the bytes that the message was deserialized from. That's pretty annoying, since it means you'd have to move the message and bytes around as a pair forever. (The reason for the lifetime is that SysEx messages contain arbitrary data as bytes, and rather than copying those bytes into the message, the MIDI library just keeps a reference to them. I assume this is to optimize for memory/speed.) If you don't care about SysEx messages (we don't), the library provides a `drop_unowned_sysex()` function that returns only the cases of MidiMessage that don't have arbitrary data like that. That returns a MidiMessage with the `'static` lifetime, since it is no longer tied to the lifetime of another object.